app/views/hackdays/_project_list.html.erb

```diff
@@ -0,0 +1,36 @@
+<% if hackday.votable %>
+  <ul>
+    <% hackday.projects.each do |proj| %>
+      <% if proj.valid? %>
```

Comment:
not clear why this is here. If the project is invalid it wouldn't have been saved in the first place.